tensorflow/python/eager/def_function_test.py

```diff
@@ -938,6 +938,17 @@ def f(x):
 
     self.assertLen(logs.output, 1)
     self.assertIn('Tracing is expensive', logs.output[0])
+  
+  def test_experimental_get_tracing_count(self):
+    @tf.function
```

Comment:
You need to use def_function.function here - def_function is just the internal namespace for tf.function.

Reply:
Thanks for the feedback, I am addressing both of your corrections and will push the changes soon. I just wonder, do you prefer me to amend my existing commits or to create new commits with my fixes?
I'd like to be compliant with the overall tensorflow style.

Reply:
New commits should be fine, they make it easier to browse the PR history. Thanks!